### ISSUE TITLE
Some way to output html comments

### ISSUE THREAD

**Issue by StokeMasterJack** [Kotlin/kotlinx.html#69]
It would be nice if we could use the DSL to generate an html comment.

**Comment by spand** [comment]
Yes that would be nice. Currently it is one of only three cases we have to use `unsafe()`.
